client/changelist/change.go

```diff
@@ -42,6 +42,20 @@ type TufRootData struct {
 	RoleName string       `json:"role"`
 }
 
+// TufDelegationUpdate represents a modification to a target delegation
+// this includes creating a delegations. This format is used to avoid
+// unexpected race conditions between humans modifying the same delegation
+type TufDelegationUpdate struct {
+	NewName            string       `json:"new_name,omitempty"`
+	NewThreshold       int          `json:"threshold, omitempty"`
+	AddKeys            data.KeyList `json:"add_keys, omitempty"`
```

Comment:
KeyList is a `[]PublicKey`, not a `[]string`, and it's explicitly defined so I can have a custom `UnmarshalJSON` that unmarshals keys into the correct types based on their algorithm.